src/api/app/models/project.rb

```diff
@@ -1550,6 +1550,14 @@ def self.very_important_projects_with_categories
     end
   end
 
+  def build_results
+    project_state.get_elements("/resultlist/result[@project='#{name}'')]")
+  end
+
+  def project_state
+    REXML::Document.new(Backend::Api::BuildResults::Status.version_releases(name))
```

Comment:
avoid REXML in any refactorings - we have 2 other xml parsers in use. This is even more important if it's unclear what is returned here in the code calling.

Reply:
Nokogiri is much faster, however to refactor the code increased some rubocop metrics score (Metrics/AbcSize, Metrics/CyclomaticComplexity and Metrics/PerceivedComplexity).. but I believe other changes that I will do in the code will be able to reduce it again